indico/modules/events/contributions/templates/display/_paper_reviewing.html

```diff
@@ -0,0 +1,112 @@
+{% from 'forms/_form.html' import form_header, form_rows, form_footer %}
+
+{% set UPLOAD_NEEDED_STATUSES = ('To be corrected', none) %}
+
+{% macro _render_paper_reviewing_status(status, icon) %}
+    <span class="{{ icon }} status-{{ status }} reviewing-status-display">
+        {{ caller() }}
+    </span>
+{% endmacro %}
+
+{% macro render_paper_files(paper_files) %}
+    <table class="tree paper-files">
+        {% for paper_file in paper_files %}
+            <tr>
+                <td>
+                    <span class="{{ icon_from_mimetype(paper_file.content_type, 'icon-file-filled') }}">
+                        <a href="{{ url_for('paper_reviewing.download_paper_file', paper_file) }}">
+                            {{ paper_file.filename }}
+                        </a>
+                    </span>
+                </td>
+                <td class="actions">
+                    <a class="icon-remove js-delete"
+                       data-method="DELETE"
+                       data-href="{{ url_for('paper_reviewing.remove_paper_file', paper_file) }}"
+                       data-title="{% trans name=paper_file.file_name %}Remove {{ name }}?{% endtrans %}"
+                       data-confirm="{% trans name=paper_file.filename -%}
+                            Are you sure you want to remove &quot;{{ name }}&quot;?<br>
+                      {%- endtrans %}"
+                      data-update=".paper-files"></a>
+                </td>
+            </tr>
+        {% endfor %}
+    </table>
+{% endmacro %}
+
+{% macro _render_upload_form(form, contribution) %}
+    {{ form_header(form, id='paper-upload-form', orientation='vertical',
+                   action=url_for('paper_reviewing.upload_paper_files', contribution)) }}
+    {{ form_rows(form, skip_labels=true) }}
+    {% call form_footer(form) %}
+        <input class="i-button big highlight" type="submit" value="{% trans %}Upload{% endtrans %}"
+               data-disabled-until-change>
+    {% endcall %}
+    <script>
+        $(document).ready(function() {
+            var dropzone = Dropzone.forElement('#paper-upload-form');
```

Comment:
`#{{ form.<fieldname>.id }}` - otherwise it'd fail if someone changed the field name or set a prefix on the form